### PR TITLE
Update editor color

### PR DESCRIPTION
**Pull request by aeng**
Update editor color to use assets.

### CODE REVIEW

File: zanata-war/src/main/java/org/zanata/webtrans/client/resources/WebTransMessages.java [hunk 1]
```diff
@@ -409,10 +409,10 @@ String undoUnsuccessful(@PluralCount int unsuccessfulCount,
     @DefaultMessage("Description")
     String description();
 
-    @DefaultMessage("Blue color")
+    @DefaultMessage("Green color")
```
Comment (djansen-redhat) [inline zanata-war/src/main/java/org/zanata/webtrans/client/resources/WebTransMessages.java]: Green equals blue?

Comment (huangp) [inline zanata-war/src/main/java/org/zanata/webtrans/client/resources/WebTransMessages.java]: This ;) Other than this :+1:

Comment (aeng) [inline zanata-war/src/main/java/org/zanata/webtrans/client/resources/WebTransMessages.java]: The diff color has been changed to Green.. but we didnt update the text.

Comment (aeng) [inline zanata-war/src/main/java/org/zanata/webtrans/client/resources/WebTransMessages.java]: I should update the method name too.

**Comment by djansen-redhat** [comment]
Colours have changed slightly, perhaps not the intention

**Comment by djansen-redhat** [comment]
:white_check_mark: Orange you glad I tested it?

**Comment by aeng** [comment]
Well, can't go pass anything without you checking..

**Comment by djansen-redhat** [comment]
Ergh, just totally disregard my pun why don't you /sarcasm

**Pull request merged**